style.css

```diff
@@ -5410,6 +5422,21 @@ h3.label.projects ~ li a {
     background: #8f8f8f;
     display: inline-block;
 } 
+	h5.projectStatus {
+		border-color: transparent;
+		border-width: 0px;
+	}
+	
+	a.liveStatus {
```

Comment:
We should add another class to the DOM: `a.projectStatus.WHATEVER-STATUS` so we can avoid repeating ourselves for the border width on these